tests/plugins/Ubuntu/Gestures/GestureTest.cpp

```diff
@@ -105,9 +105,6 @@ void GestureTest::sendTouch(qint64 timestamp, int id, QPointF pos,
 
     QTouchEvent touchEvent(eventType, m_device, Qt::NoModifier, Qt::TouchPointPressed, points);
     QCoreApplication::sendEvent(m_view, &touchEvent);
-
-    QQuickWindowPrivate *windowPrivate = QQuickWindowPrivate::get(m_view);
-    windowPrivate->flushDelayedTouchEvent();
```

Comment:
These changes might bring in a race condition where not all events have been delivered before the next comparison which assumes they have.

Looks like the api has changed, but [`flushFrameSynchronousEvents`](https://code.woboq.org/qt5/qtdeclarative/src/quick/items/qquickwindow.cpp.html#_ZN19QQuickWindowPrivate27flushFrameSynchronousEventsEv) should do the same thing and ensure all events have been delivered.